blaze/compute/pandas.py

```diff
@@ -165,6 +165,21 @@ def compute_up(expr, data, **kwargs):
     return compute_up(expr._child.distinct().count(), data, **kwargs)
 
 
+@dispatch(Repeat, Series)
+def compute_up(expr, data, **kwargs):
+    return expr.lhs.str.repeat(data)
```

Comment:
similar to `Concat` i think this implementation is unnecessary because of subclassing